backend/Recodo.BLL/Services/AuthService.cs

```diff
@@ -64,5 +70,23 @@ public async Task<TokenDTO> GenerateAccessToken(int id, string userName, string
             return new TokenDTO(accessToken, refreshToken);
         }
 
+        public async Task<GoogleJsonWebSignature.Payload> VerifyGoogleToken(ExternalAuthDto externalAuth)
+        {
+            try
+            {
+                string clientId = _configuration["GoogleClientId"];
+                var settings = new GoogleJsonWebSignature.ValidationSettings()
```

Comment:
Is it possible to configure it once on the start of the application (I suppose there should be an extension for AddAuthentication() in Startup class)

Reply:
It's just a class with a field "Audience" that is passed to ValidateAsync. It's not settings